packages/apps/tools/src/pages/faucet/new/index.tsx

```diff
@@ -293,13 +297,11 @@ const NewAccountFaucetPage: FC = () => {
               <div className={inputWrapperStyle}>
                 <PublicKeyField
                   helperText={errors?.pubKey?.message}
-                  inputProps={{
-                    ...register('pubKey', {
-                      onChange: () => {
-                        clearErrors('pubKey');
-                      },
-                    }),
-                  }}
+                  {...register('pubKey', {
+                    onChange: () => {
```

Comment:
you can remove this and use `mod: 'onChange'` for the `useForm` hook instead

Reply:
We can't, the `clearErrors` needs to be called inside `onChange` since we manually call `setError` elsewhere. So it's correct as it is 😁

Reply:
in this case, I would recommend using a custom validation function instead of manually setting the errors, you can use zod refinements for that purpose https://zod.dev/?id=refine

Reply:
but I said in the other comments it is not a concern for this PR, and we can improve or keep as it is later if it is needs that